superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -67,6 +67,13 @@ const CONFIRM_OVERWRITE_MESSAGE = t(
     'sure you want to overwrite?',
 );
 
+const OVERWRITE_FIELDS: string[] = [
+  'dashboards',
+  'charts',
+  'datasets',
+  'databases',
+];
```

Comment:
See my comment below, we might want to have these in the singular form to align with the `ImportModelsCommand.model_name` attribute.

Reply:
OK got it, good idea, I will change it to a singular form